src/SamplesApp/SamplesApp.UITests.Generator/Helper/SourceGenerationHelper.cs

```diff
@@ -1,23 +0,0 @@
-﻿using Microsoft.CodeAnalysis;
-
-namespace Uno.Samples.UITest.Generator.Helper
-{
-	internal static class GeneratorExecutionContextExtensions
-	{
-		private const string SourceItemGroupMetadata = "build_metadata.AdditionalFiles.SourceItemGroup";
-
-		public static string GetMSBuildPropertyValue(
-			this GeneratorExecutionContext context,
-			string name,
-			string defaultValue = "")
-		{
-			context.AnalyzerConfigOptions.GlobalOptions.TryGetValue($"build_property.{name}", out var value);
-			return value ?? defaultValue;
-		}
-
-		public static bool TryGetOptionValue(this GeneratorExecutionContext context, AdditionalText textFile, string key, out string? value)
-		{
-			return context.AnalyzerConfigOptions.GetOptions(textFile).TryGetValue(key, out value);
-		}
-	}
-}
```

Comment:
These helpers already exist. They are added in the new shared project and are shared between `Uno.UI.SourceGenerators` and `SamplesApp.UITests.Generator`